dist/images/Dockerfile.fedora.dev

```diff
@@ -39,7 +39,7 @@ ARG OVS_BRANCH=master
 
 RUN dnf upgrade -y && dnf install --best --refresh -y --setopt=tsflags=nodocs \
 	PyYAML bind-utils procps-ng openssl numactl-libs firewalld-filesystem \
-        libpcap hostname kubernetes-client \
+        libpcap hostname kubernetes-client python3-openvswitch\
         iptables iproute iputils strace socat\
         "kernel-devel-uname-r == $KERNEL_VERSION" \
```

Comment:
@danielmellado for sake of consistency can you please add a `space` before `\` at L42 and 43, please. thanks.

Reply:
done! thanks for noticing! ;)